src/coverlet.core/Instrumentation/Instrumenter.cs

```diff
@@ -269,7 +270,10 @@ private static void ReplaceExceptionHandlerBoundary(ExceptionHandler handler, In
 
         private static bool IsExcludeAttribute(CustomAttribute customAttribute)
         {
-            return customAttribute.AttributeType.Name == nameof(ExcludeFromCoverageAttribute) || customAttribute.AttributeType.Name == "ExcludeFromCoverage";
+            var attributeName = customAttribute.AttributeType.Name;
+
+            return attributeName == nameof(ExcludeFromCoverageAttribute) || attributeName == "ExcludeFromCoverage"
+                || attributeName == nameof(ExcludeFromCodeCoverageAttribute) || attributeName == "ExcludeFromCodeCoverage";
```

Comment:
It's probably better to have the attribute names in an array and do a simple `array.Any(...)`